java/ingest/ingest-runner/src/test/java/sleeper/ingest/runner/IngestRowsIT.java

```diff
@@ -542,7 +544,7 @@ void shouldSortRows() throws Exception {
     }
 
     @Test
-    void shouldApplyIterator() throws Exception {
+    void shouldApplyAdditionIterator() throws Exception {
```

Comment:
This test isn't about the addition iterator. The intention is that you can set any generic iterator if you refer to it by its class name. We can say that in the test name. Previously that was the only way you could set an iterator, but it's not the only way any more.

Expanding on my previous comment: https://github.com/gchq/sleeper/pull/5519#discussion_r2306718748